f1pystats/f1pystats.py

```diff
@@ -208,6 +211,22 @@ def finishing_status(year: int, race_round: int = 0):
         zip(status_id, status_info, status_count), columns=["StatusId", "Status", "Count"]
     )
 
+def sprint_results(year: int, race_round: int):
+    """Returns the sprint qualifying results for a year and round"""
+    json_data=_get_json_content_from_url(
+        f"http://ergast.com/api/f1/{year}/{race_round}/sprint.json"
+    )
+    if len(json_data["MRData"]["RaceTable"]["Races"])==0 :
+        return "No Sprint Race found for this Grand Prix"
```

Comment:
I think it would be neater to throw an exception instead of returning a string.

Perhaps you can change this to something like:
```
raise Exception("ERR: No Sprint Race found for this Grand Prix")
```